### DISCUSSION TITLE
1 BRC - A beginner's Guide

### DISCUSSION THREAD

**Discussion by mishra-kunal1** [gunnarmorling/1brc#714] [Show and tell]
https://mishra-kunal1.github.io/1brc/
I was exploring the solutions in Java, Python, and C++. I have written a blog to explain the code behind a few of the solutions. 
Let me know your reviews about it.

**Comment by lprakashv** [comment]
I think there is some inconsistency in blog's publish date "February 01, 2020", because the challenge was started in 2024 :)
